docs/HowToUseSmartFS.md

```diff
@@ -73,11 +74,27 @@ void s5j_board_initialize(void)
 #endif
 }
 ```
```

Comment:
You can use syntax highlighter like this.
````
```c
/* c code */
```
````



Highlighten version
```c
/* In case of imxrt */
void __start(void)
{
	...
	/* Initialize onboard resources */
	imxrt_boardinitialize();
	...
}
```

Un-highlighten version
```
/* In case of imxrt */
void __start(void)
{
	...
	/* Initialize onboard resources */
	imxrt_boardinitialize();
	...
}
```

Reply:
Oh, thank you for useful information.
Do we need to use ```c rather than ``` for syntax highlighting?

Reply:
Syntax highlighting improves readability, and a good guide is easy to read.
So, I think syntax highlighting should be used in code snippet.
How do you think?

Reply:
I fully agree with you. Applied.